.changeset/beige-bananas-divide.md

```diff
@@ -0,0 +1,15 @@
+---
+'@sap-cloud-sdk/connectivity': minor
+---
+
+[New Functionality] Add convenience function `transformServiceBindingToDestination` to create OAuth2ClientCredentials destination from service bindings.
```

Comment:
[req] According to our styleguide functions should have parantheses in documentation aka. `transformServiceBindingToDestination()`